index.js

```diff
@@ -9,6 +9,22 @@ const TSCONFIG_PATH = process.env.TSCONFIG_PATH
 
 const { SourceMapConsumer, SourceMapGenerator } = require('source-map')
 
+function loadJsonFile(jsonFilename) {
+  if (!fs.existsSync(jsonFilename)) {
+    throw new Error(`Input file not found: ${jsonFilename}`)
+  }
+
+  const buffer = fs.readFileSync(jsonFilename)
+  try {
+    let jju = require('jju')
+    return jju.parse(buffer.toString())
+  } catch (error) {
+    throw new Error(
+      `Error reading "${jsonFilename}":` + os.EOL + `  ${error.message}`
```

Comment:
Why is os.EOL concatenated with + here rather than using the interpolation syntax?